content/events-api/limits-and-scalability.md

```diff
@@ -2,8 +2,6 @@
 
 ## PIM side limits
 
-To not degrade the performance of the PIM certain limitations can be configured.
-
 ### Number of event subscriptions
 
 By default, you can only subscribe to `3` events. To change this behavior update the value of the parameter `webhook_active_event_subscriptions_limit` in the `config/services/pim_parameters.yml` file.
```

Comment:
```suggestion
You can only subscribe to `3` events.
```

Reply:
We would need to do that for every limit specified.

Reply:
I'm ok to update the documentation by removing all the set-up possibilities for Serenity customers. Do we need to have a dedicated page for PaaS customers?

Reply:
This kind of documentation (changing code or settings) is usually done in the technical doc docs.akeneo.com. We can do that after.